src/lib/containers/EntityLink.tsx

```diff
@@ -78,16 +78,16 @@ export const EntityLink: React.SFC<EntityLinkProps> = ({
 }) => {
   const { id, name, type } = entityHeader
   return (
-    <a
-      className={className}
-      target="_blank"
-      rel="noopener noreferrer"
-      href={`https://www.synapse.org/#!Synapse:${id}`}
-    >
-      <p className={className}>
-        <EntityTypeIcon type={type} />
-        {name}
-      </p>
-    </a>
+    <p className={className}>
```

Comment:
This reorg changes the target (on the click event) to the actual anchor, where I can interrogate the href.